### PR TITLE
Added option to automatically determine port to WebRtcServer

### PR DESCRIPTION
**Pull request by satoren**
If port is not specified at createWebRtcServer, automatically determine the port of the webrtcserver by Worker's port range

https://mediasoup.discourse.group/t/can-webrtcserver-determine-the-port-automatically/4544/6

### CODE REVIEW

**Comment by nazar-pc** [comment]
Same thing I commented on original PR :slightly_smiling_face:  https://github.com/versatica/mediasoup/pull/834#issuecomment-1166606629

File: worker/src/RTC/WebRtcServer.cpp
Comment (ibc) [inline worker/src/RTC/WebRtcServer.cpp:120]: Please blank line before the if condition.

File: worker/src/RTC/WebRtcServer.cpp
Comment (ibc) [inline worker/src/RTC/WebRtcServer.cpp]: These conditions are a bit hard to read. We usually handle first the case in which value is properly given, then lack of given value (so ser port to 0), then the throw due to invalid value.

**Review by ibc — COMMENTED** [review]
Just cosmetic changes required.

File: worker/src/RTC/WebRtcServer.cpp
Comment (ibc) [inline worker/src/RTC/WebRtcServer.cpp:108]: Please blank line after }

File: worker/src/RTC/WebRtcServer.cpp
Comment (ibc) [inline worker/src/RTC/WebRtcServer.cpp:130]: Blank line after this line as above, please

**Review by ibc — CHANGES_REQUESTED** [review]

**Comment by ibc** [comment]
Lint or something is failing in CI

**Review by jmillan — APPROVED** [review]

**Comment by ibc** [comment]
I won't merge it yet since I want to write doc in mediasoup-website project before merging it. If you wish to make a PR to the docs project, you are welcome. Otherwise I'll do hopefully this week.

**Review by ibc — APPROVED** [review]

**Comment by ibc** [comment]
Released in 3.10.8

**Pull request merged**
